smarts/core/vehicle_index.py

```diff
@@ -653,11 +653,13 @@ def build_agent_vehicle(
         surface_patches,
         initial_speed=None,
         boid=False,
+        *,
```

Comment:
Is this a typo?

Reply:
The single star * syntax is from [PEP3102](https://peps.python.org/pep-3102/).

Reply:
I'm not sure I understand why `vehicle_id` needs to be a "keyword-only argument".

Reply:
It does not need to be 'keyword-only' but it ensures that the value is not specified accidentally.

I have a preference for keeping it 'keyword-only' but no heavy attachments.